.g8/radioButtonPage/app/controllers/$className$Controller.scala

```diff
@@ -61,7 +62,7 @@ class $className;format="cap"$Controller @Inject()(
 
 object $className;format="cap"$Controller {
 
-  def form(formProvider: RadioListFormProvider) =
+  def form(formProvider: RadioListFormProvider): Form[???] =
```

Comment:
Are .g8 changes intentionally pushed?

Reply:
yeah, I just tweaked them to not fail compilation when used